src/test/java/org/broadinstitute/hellbender/utils/svd/SingularValueDecomposerUnitTest.java

```diff
@@ -31,11 +31,7 @@ public void testBasicFunction(final SingularValueDecomposer svder){
         final double[] expectedSingularValues = {3.0, 2.0};
 
         final SVD svd = svder.createSVD(m);
-        Assert.assertEquals(svd.getU().multiply(MatrixUtils.createRealDiagonalMatrix(svd.getSingularValues())).multiply(svd.getV().transpose()), m);
-        Assert.assertTrue(isUnitaryMatrix(svd.getV()), "V is unitary");
-        Assert.assertTrue(isUnitaryMatrix(svd.getU()), "U is unitary");
-        Assert.assertEquals(svd.getV().transpose().multiply(svd.getV()), MatrixUtils.createRealIdentityMatrix(2), "V is unitary");
-        Assert.assertEquals(svd.getU().transpose().multiply(svd.getU()), MatrixUtils.createRealIdentityMatrix(2), "U is unitary");
+        assertSVD(svd, m);
```

Comment:
hmm, that may be true in some, but not in the current implementation of SVD in GATK. U may not be square, as per doc for apache http://commons.apache.org/proper/commons-math/apidocs/org/apache/commons/math3/linear/SingularValueDecomposition.html
and mllib https://spark.apache.org/docs/1.2.0/mllib-dimensionality-reduction.html

U is not square in this unit test in this class
M={{2.0,0.0},{0.0,-3.0},{0.0,0.0}}

U = {{0.0,-1.0},{-1.0,0.0},{0.0,0.0}}
S = {{3.0,0.0},{0.0,2.0}}
V = {{0.0,-1.0},{1.0,0.0}}